astropy/io/misc/ecsv.py

```diff
@@ -192,12 +195,16 @@ class ECSVHeader:
     delimiter: str
 
 
-class ECSVEngine:
+class ECSVEngine(metaclass=abc.ABCMeta):
     """Base class for ECSV reader engines.
 
     An engine is responsible for reading the raw CSV data that follows the ECSV header.
     This assumes that the engine has a defined Table Unified I/O interface.
 
+    - ``name`` and ``format`` must be defined as class attributes in subclasses.
```

Comment:
Does this not link to the Properties section if made single?

```suggestion
    - `name` and `format` must be defined as class attributes in subclasses.
```